### PR TITLE
added EventHelper module

### PR DESCRIPTION
**Pull request by jtkpiotr**
@quirk0o @lkudyba @mhib @pbzymek @adamzima @s1mplex

### CODE REVIEW

File: client/test/factories/Event.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import ConferenceRoom from './ConferenceRoom';
 export default new Factory()
   .sequence('id')
   .option('start_time', new Date(2016, 7, 25, 0, 0, 0))
-  .option('end_time', new Date(2016, 7, 25, 2, 0, 0))
+  .option('end_time', ['start_time'], startTime => new Date((new Date()).setHours(startTime.getHours() + 4)))
```
Comment (lkudyba) [inline client/test/factories/Event.js]: I suggest using moment.js: `moment(startTime).add(4, 'hours')`

File: client/test/helpers/EventHelper.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { expect } from 'chai';
+import Event from '../factories/Event';
+
+describe('EventHelper', () => {
+  describe('#eventStartsAt()', () => {
+    const timestamp = (new Date()).getTime() / 1000;
+    const event = Event.build();
+
+    it('returns a function', () => {
+      expect(EventHelper.eventStartsAt(timestamp)).to.be.an('function');
```
Comment (lkudyba) [inline client/test/helpers/EventHelper.spec.jsx]: use `to.be.a` for 'function'

File: client/test/helpers/EventHelper.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  });
+  describe('#groupInDay()', () => {
+    const event1 = Event.build({}, { start_time: new Date(2016, 7, 25, 4, 0, 0) });
+    const event2 = Event.build({}, { start_time: new Date(2016, 7, 25, 12, 0, 0) });
+    const event3 = Event.build({}, { start_time: new Date(2016, 7, 26, 4, 0, 0) });
+    const event4 = Event.build({}, { start_time: new Date(2016, 7, 26, 12, 0, 0) });
+    const timestamp = (new Date(2016, 7, 25, 0, 0, 0)).getTime() / 1000;
+    const group1 = [event1, event2];
+    const group2 = [event3, event4];
+    it('returns true if given group of events sorted by starting time belongs to a specific day', () => {
```
Comment (lkudyba) [inline client/test/helpers/EventHelper.spec.jsx]: use more concise test names

File: client/app/helpers/EventHelper.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+export const SECONDS_IN_DAY = 24 * 60 * 60;
+
+export function eventStartsAt(timestamp) {
+  return (event) => event.start_timestamp === timestamp;
+}
+
+export function groupInDay(group, timestamp) {
```
Comment (madejejej) [inline client/app/helpers/EventHelper.jsx]: Do you need to export this function?

File: client/app/helpers/EventHelper.jsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+export const SECONDS_IN_DAY = 24 * 60 * 60;
+
+export function eventStartsAt(timestamp) {
```
Comment (madejejej) [inline client/app/helpers/EventHelper.jsx]: Do you need to export this function?

File: client/test/helpers/EventHelper.spec.jsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+  describe('#eventsStartingAt()', () => {
+    const event1 = Event.build({}, { start_time: new Date(2016, 7, 25, 0, 0, 0) });
+    const event2 = Event.build({}, { start_time: new Date(2016, 7, 25, 0, 0, 0) });
+    const event3 = Event.build({}, { start_time: new Date(2016, 7, 25, 1, 0, 0) });
+    const group = [event1, event2, event3];
+    const timestamp = (new Date(2016, 7, 25, 0, 0, 0)).getTime() / 1000;
+    it('returns events that starts at the specific time', () => {
+      expect(EventHelper.eventsStartingAt(timestamp, group)).to.include(event1, event2);
+      expect(EventHelper.eventsStartingAt(timestamp, group)).not.to.include(event3);
```
Comment (quirk0o) [inline client/test/helpers/EventHelper.spec.jsx]: You can use `.to.eql([event1, event2])` to check for deep array equality

**Pull request merged**